src/main/java/haveric/recipeManager/tools/Version.java

```diff
@@ -52,6 +55,19 @@ private static boolean supports1_11() {
 
         return supports;
     }
+    private static boolean supports1_10() {

```

Comment:
I guess it's a gutcheck thing. Personally, I'm moving off of 1.10 and probably won't spend much time looking back. I know some folks in my "sphere" of admins are probably going to stay on 1.10 and use RM a lot, but I don't think they've encountered any issues in this arena.

So, it'd probably just end up being crufty.